RevPiDevice.c

```diff
@@ -142,7 +161,7 @@ void RevPiDevice_finish(void)
 int RevPiDevice_run(void)
 {
 	INT8U i8uDevice = 0;
-	INT32U r;
+	INT32U r = 0;
```

Comment:
This is removed again in the next commit of this pull request. Please dont set it at the first place